### ISSUE TITLE
Training process  (and congrats TPAMI accepting) 

### ISSUE THREAD

**Issue by damonchang23** [HongwenZhang/PyMAF-X#15]
First of all, congrats accept TPAMI your great work.
I want to training your network from sketch for research purpose but there is no training precedure or details in your README.md.
If you don't mind, can you guide how to train your network from sketch?

Thank you for your time and help.

**Comment by HongwenZhang** [comment]
Hi, sorry for the slow update of this project.

To train the code, the most tedious part is the preparation of the training data.
I have just uploaded the compatible [SMPL-X labels](https://cloud.tsinghua.edu.cn/d/a661f9acb537465fbc8d/), and hope these files could help.

The example script to run the training code:
`python -m apps.train --regressor pymaf_net --train_data h36m_coco_itw --pretrained_checkpoint xxxx --misc TRAIN.BATCH_SIZE 64 TRAIN.NUM_WORKERS 8 LOSS.SHAPE_W 0.6 MODEL.PyMAF.BACKBONE hr48`

**Comment by fatbao55** [comment]
Hi @HongwenZhang, thanks for the great work! Do you also mind sharing your training process and data preparation for the AGORA benchmark?

**Comment by ruowenyu** [comment]
Hi, @damonchang23, I also wand to train PyMAF-X from scatch, but encounter some issues during training. I was wondering if you train PyMAF-X successfully? Thanks for the help!
